Libplanet/Store/IStore.cs

```diff
@@ -27,6 +27,17 @@ public interface IStore
 
         bool DeleteIndex(string @namespace, HashDigest<SHA256> hash);
 
+        /// <summary>
+        /// Deletes an index, tx nonce and state references in the given
```

Comment:
How about adding a comma here for consistency?
```suggestion
        /// Deletes an index, tx nonces, and state references in the given
```